Geometry/CaloEventSetup/plugins/CaloGeometryBuilder.h

```diff
@@ -51,5 +52,13 @@ class CaloGeometryBuilder : public edm::ESProducer
       // ----------member data ---------------------------
       
       std::vector<std::string> theCaloList;
```

Comment:
```caloList_```?

Reply:
This one was actually a leftover from the change in approach, so I removed it altogether.